os/tools/mksyscall.c

```diff
@@ -482,15 +485,20 @@ static void generate_stub(int nparms)
 	/* Tail end of the function.  If the proxied function has no return
 	 * value, just return zero (OK).
 	 */
-
-	if (strcmp(get_parm(RETTYPE_INDEX), "void") == 0)
-		fprintf(stream, ");\n  return 0;\n}\n\n");
-	else
-		fprintf(stream, ");\n}\n\n");
-
-	cond_parm = get_parm(COND_INDEX);
-	if (cond_parm[0] != '\0')
-		fprintf(stream, "#endif /* %s */\n", cond_parm);
+	if(cond_parm[0] == '\0'){
```

Comment:
Please check coding rules. 
Space required between `)` and `{`
Space between `if` and `(`
Please check coding rule violations in all changed you have made.